community/pmc-tool/main/views.py

```diff
@@ -1,64 +1,11 @@
 from django.shortcuts import render
-from main.collector.db import create_db_connection, create_pull_requests_table, insert_pull_requests_to_db, \
-    fetch_pull_requests, create_issues_table, fetch_issues, insert_issues_to_db
-from main.collector.gh import query_prs, query_issues
-import logging
+from main.collector.data_processor import save_data_to_db
 
 
-def assert_equal(db_data, api_data):
-    db_set = set(db_data)
-    api_set = set(api_data.items())
-    if db_set != api_set:
-        raise ValueError('The data from the database does not match the data from the API. '
-                         + 'Database has %s extra. ' % (db_set - api_set)
-                         + 'API has %s extra.' % (api_set - db_set))
+def render_results(request):
+    pr_results, issue_results, first_pr_authors, new_authors = save_data_to_db()
```

Comment:
In this case, for `render_results`, we are still calling `save_data_to_db`, which makes HTTP requests AND makes Database queries. We do not want that.

Like I said, if we are making HTTP requests at the same time as we render the page, our page will take several seconds to render. That's why we want a separate function that makes HTTP requests to github that will NOT run when we render the page. When we render the page we only want to perform database queries.

Why? 
- HTTP requests to github take about 1 second each. We will take minimum 5 seconds to fetch all data from Github.
- Database queries take about 50 milliseconds, which is acceptable latency when we are rendering a page.

We want:
- `save_data_to_db` as an independent function that ONLY makes HTTP requests to github and stores that data in SQLLite
- Another function, perhaps `read_reports` that ONLY makes SELECT queries from SQLite, but does not make connections to github.